gerrit/auth.go

```diff
@@ -89,3 +116,17 @@ func (r *authManager) cleanup() {
 		r.cookiesPath_ = ""
 	}
 }
+
+func (r *credentialsAuth) GerritAuth() (gerrit.Auth, error) {
+	return gerrit.BasicAuth(r.username, r.password), nil
```

Comment:
Have you tried DigestAuth instead of BasicAuth here? If it works it is quite a bit more secure.

Reply:
I didn't, I'll replace basic authentication with digest